packages/stream_chat_localizations/lib/src/stream_chat_localizations_ko.dart

```diff
@@ -344,4 +344,7 @@ class StreamChatLocalizationsKo extends GlobalStreamChatLocalizations {
 
   @override
   String get replyToMessageLabel => '메시지에 회신합니다.';
+
+  @override
+  String get slowModeOnLabel => '느린 모드 켜짐';
```

Comment:
```suggestion
  String get slowModeOnLabel => '슬로모드온 입니다.';
```
@EricSSartorius thoughts?

Reply:
슬로모드 켜짐 Might be best here

Reply:
```suggestion
  String get slowModeOnLabel => '슬로모드 켜짐';
```